pandas/tseries/tools.py

```diff
@@ -229,13 +229,28 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         - If False, allow the format to match anywhere in the target string.
 
     unit : string, default 'ns'
-        unit of the arg (D,s,ms,us,ns) denote the unit in epoch
-        (e.g. a unix timestamp), which is an integer/float number.
+        unit of the arg (D,s,ms,us,ns) denote the unit, which is an
+        integer or float number. This will be based off the origin.
+        Example, with unit='ms' and origin='unix' (the default), this
+        would calculate the number of milliseconds to the unix epoch start.
     infer_datetime_format : boolean, default False
         If True and no `format` is given, attempt to infer the format of the
         datetime strings, and if it can be inferred, switch to a faster
         method of parsing them. In some cases this can increase the parsing
         speed by ~5-10x.
+    origin : scalar convertible to Timestamp / string ('julian', 'unix'),
+        default 'unix'.
```

Comment:
The problem is that this does not work ... type explanation should be on one line (for good html docs) (ref https://github.com/numpy/numpydoc/issues/87)